src/react-components/avatar-selector.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage } from 'react-intl';
+
+class AvatarSelector extends Component {
+  static propTypes = {
+    avatars: PropTypes.array,
+    avatar: PropTypes.string,
+    onChange: PropTypes.func,
+  }
+
+  getAvatarIndex(direction=0) {
```

Comment:
should this be a `= () => {`? looks like this function assumes consistent `this`

Reply:
Sure, I guess it improves consistency.